charts/redis-ha/Chart.yaml

```diff
@@ -5,7 +5,7 @@ keywords:
 - redis
 - keyvalue
 - database
-version: 4.26.6
```

Comment:
That's quite a jump? CI might reject with this now being an increment of 1.

Reply:
Ohh :D sorry, my bad. I fixed it